CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Changelog
+---
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to [Semantic Versioning](https://semver.org/).
+
+## [Unreleased]
+
+Just minor documentation fixes.
```

Comment:
Then we should write this up as a release

Reply:
I wanted to fix the styles issue as well before releasing a new version